ROADMAP.md

```diff
@@ -48,7 +61,8 @@ The list is sorted by priority
 
 * [x] C
 * [x] C++
-* [ ] Python
+* [x] Python
+* [ ] Go
 * [ ] Swift
 * [ ] Kotlin
```

Comment:
Only if you have to create another meaningful commit, else this can be done later. Could you add Dart and sort the unimplemented language bindings?